### PR TITLE
Coral-Service: Modify README to add Coral-Service UI description

### PR DESCRIPTION
**Pull request by ljfgem**
This PR adds Coral-Service UI description to README, and also removes the unnecessary `index` method from controller, which can be handled by Thymeleaf.

And the UI can be used for both local and remote metastore mode.

Please check [the rendered README](https://github.com/ljfgem/coral/tree/modify_readme_to_include_ui#coral-service-ui) for convenient review.

cc: @wmoustafa @aastha25

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -147,6 +145,32 @@ You may also achieve above 5-7 via Coral-Service simple UI: http://localhost:808
 ```  
 7. Translate a query on existing table/view in remote metastore using the /translate endpoint
 
+### Coral Service UI
+Apart from the CLI command above, you can also interact with Coral Service via a simple UI.
+
+After running `../gradlew bootRun` or `../gradlew bootRun --args='--spring.profiles.active=localMetastore'` in coral-service module, 
+the UI can be accessed by http://localhost:8080:
```
Comment (wmoustafa) [inline README.md:152]: After running `../gradlew bootRun` (for remote metastore mode) or `../gradlew bootRun --args='--spring.profiles.active=localMetastore'` (for local metastore mode) from coral-service module, 
the UI can be accessed from the browser. Use the URL `http://localhost:8080:` to run the UI on a local browser.

**Comment by wmoustafa** [comment]
Could you make the create table command in the screenshot uppercase?

Also some build checks are failing.

**Comment by wmoustafa** [comment]
I have also noticed the box stating the translation result mentions `spark`, `trino`, etc. Would be great to use `Spark SQL`, `Trino SQL`, etc.

File: README.md (hunk 1 above)
Comment (ljfgem) [reply README.md]: Modified, thanks.

**Review by wmoustafa — COMMENTED** [review]
Thanks @ljfgem!

**Comment by aastha25** [comment]
LGTM, thanks @ljfgem

**Review by wmoustafa — APPROVED** [review]

**Pull request merged**